src/app/pricing/page.tsx

```diff
@@ -46,37 +51,147 @@ export default function PricingPage() {
   }
 
   return (
-    <div className="PricingPage flex w-[75%] max-w-[1600px] flex-col self-center p-14 pb-4">
+    <div className="PricingPage flex w-[95%] max-w-[1600px] flex-col self-center p-14 pb-4 sm:w-[75%]">
       <div className="PricingTitleContainer flex flex-col self-start text-left">
         <h1 className="PricingTitle">our straightforward pricing:</h1>
-        <p className="PricingSubtitle italic">
+        <p className="PricingSubtitle mt-8 w-[70%] self-end text-right text-sm italic sm:w-full sm:text-left sm:text-lg md:mt-4">
           {`because that's how all pricing should be`}
         </p>
       </div>
-      <div className="PricingOptions flex gap-12 self-center pt-12">
+      <div className="PricingOptions flex flex-col gap-8 self-center pt-12 sm:flex-row sm:gap-12">
         <SiteButton
           aria="features for businesses"
           size="large"
           variant="filled"
           colorScheme="b3"
-          addClasses="px-14"
+          addClasses="px-14 w-[20rem]"
           onClick={() => handleClick("business")}
         >
           for businesses
         </SiteButton>
+
+        {/* business pricing details - MOBILE*/}
```

Comment:
The way you render this between the site buttons makes it render oddly on table (see 695px width)